src/rtcModule/webrtcPrivate.h

```diff
@@ -301,6 +304,8 @@ class Call: public ICall
     void updateAvFlags(karere::Id userid, uint32_t clientid, karere::AvFlags flags);
     bool isCaller(karere::Id userid, uint32_t clientid);
     void changeVideoInDevice();
+    virtual bool isAudioMonitorEnabled() const override;
```

Comment:
```suggestion
    bool isAudioLevelMonitorEnabled() const override;
```
`virtual` should not be here, but `override`